doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -127,7 +130,7 @@ Additional scenario XML elements:
 ``clos``:
   Class of Service for Cache Allocation Technology. Refer to the SDM 17.19.2 for details and use with caution.
```

Comment:
Why "use with caution"? Can we add (a few) simple words as to what to pay attention to?